logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/DefaultMinLogLevelIT.java

```diff
@@ -25,4 +43,33 @@ public void checkDefaultLogMinLevel() {
         // the value of minimum logging level overrides the logging level
         app.logs().assertDoesNotContain("Trace log example");
     }
+
+    // Verifies https://github.com/quarkusio/quarkus/issues/40016
+    @Test
+    public void checkLogRotationContent() throws IOException {
+        Path logDir = app.getServiceFolder();
+
+        app.given().when().get("/log").then().statusCode(204);
+
+        Awaitility.await().atMost(10, TimeUnit.SECONDS).until(() -> {
+            File[] rotatedFiles = logDir.toFile().listFiles((dir, name) -> name.startsWith(LOG_FILE_NAME));
+            return rotatedFiles != null && rotatedFiles.length > 1;
```

Comment:
yeah, so, you expect 3 files but are content with finding one? I understood your comment that you want to find this line in 3 files, so I expect you need 3 files. Won't the test be flaky if you wait just for one and hope others are stored in the meanwhile?

Reply:
Tbh I don't think it was wrong as with backup logs you can't know how many will be there. Someone can update `quarkus.log.file.rotation.max-file-size` and this test can have only one backup log.

Reply:
sorry, that is not acceptable; someone cannot just update `max-file-size` because this is part of the test; if he updates it, the test must fail

Reply:
@jedla97 configuration properties are either required for test in their current value to test a feature, or we should drop them

Reply:
Sure, I have no idea whether this test is stable. I commented on changing property for no reason. by looking at this test I think it is weird I don't understand why it is separated to 3 files and max-file-size refers to memory size, not lines. I somehow expected @gtroitsk have it worked out :-)